client/lib/domains/tlds/wpcom-tlds.json

```diff
@@ -0,0 +1,268 @@
+{
+    "academy": 1,
+    "accountants": 1,
+    "actor": 1,
+    "agency": 1,
+    "airforce": 1,
+    "apartments": 1,
+    "army": 1,
+    "associates": 1,
+    "attorney": 1,
+    "auction": 1,
+    "band": 1,
+    "bargains": 1,
+    "be": 0,
+    "bike": 1,
+    "bingo": 1,
+    "biz": 1,
+    "blog": 1,
+    "boutique": 1,
+    "builders": 1,
+    "business": 1,
+    "ca": 1,
+    "cafe": 1,
+    "camera": 1,
+    "camp": 1,
+    "capital": 1,
+    "cards": 1,
+    "care": 1,
+    "careers": 1,
+    "cash": 1,
+    "casino": 1,
+    "catering": 1,
+    "center": 1,
+    "chat": 1,
+    "cheap": 1,
+    "church": 1,
+    "city": 1,
+    "claims": 1,
+    "cleaning": 1,
+    "clinic": 1,
+    "clothing": 1,
+    "cloud": 1,
+    "club": 1,
+    "co": 1,
+    "coach": 1,
+    "codes": 1,
+    "coffee": 1,
+    "co.in": 1,
+    "com": 1,
+    "com.br": 0,
```

Comment:
I'm curious about these zeros. Do they mean anything? We just check for not `undefined` in parseDomainAgainstTldList, right?

Reply:
Right, enabled/disabled data in case anyone consuming the list were interested.